### ISSUE TITLE
Are EAP builds free for commercial use?

### ISSUE THREAD

**Issue by ArrayBolt3** [BashSupport-Pro/bashsupport-pro#148]
Currently I'm working a job where I do a *lot* of Bash coding (both for closed-source and open-source code). Currently we aren't in a good position to purchase a BashSupportPro license for me (though that may change in the future).

I noticed that BashSupportPro has an Early Access Program, whereby I can use it (presumably for free?) in exchange for testing pre-release, potentially unstable features and reporting bugs as I find them. I frequently use pre-release software as part of my paid and volunteer jobs in open-source work, and would be comfortable using potentially unstable software like this. I also generally try to write good-quality bug reports and assist in bug triage (I have to deal with bug reports myself and have come to appreciate good ones).

Are the EAP builds of BashSupportPro free? And if so, is it acceptable to use them for commercial use? If not, I'm happy to just stick with editing in Vim and buy a license later.

**Comment by jansorg** [comment]
@ArrayBolt3 EAPs are free to use for personal and commercial use. But they'll expire after 30 days and still need a trial or active license. The requirement for a license to use EAPs will change in the future, but at this time it's still there due to technical limitations.
The final 4.0 release will be out in a few days.

Please note that it's possible to use a personal license (purchases with your own funds) at work, but that's probably not what you'd like. 
There are free licenses for open-source work, refer to https://www.bashsupport.com/pricing/ for details.
Please let me know if you still have questions about licensing or the software. 
Thanks!

**Comment by ArrayBolt3** [comment]
Thanks for your help! Unfortunately it appears that since I already used the trial license, I can't activate the EAP build. (I used up the whole 30 day trial period already, which got me hooked :P) That's fine, I'll just wait until we can afford a license and then purchase one.

Thanks for the pointer to the open-source license work. I don't think I can use those since my work is not entirely open-source, and much of my open-source work is commercial in nature. I suppose I *could* apply for an open-source license (since I do a lot of volunteer open-source work too) and then use that to work around the technical limitations of JetBrains Marketplace so I could use the EAP, but that feels wrong.

Anyway, in a few weeks here I may be able to afford a license more easily. Thanks for your help and for the making this awesome tool, and have a great day!
